walkthroughs/week-5-tdd/project/src/test/java/com/google/sps/FindMeetingQueryTest.java

```diff
@@ -270,5 +272,149 @@ public void notEnoughRoom() {
 
     Assert.assertEquals(expected, actual);
   }
+
+  @Test
+  public void busyOptionalAttendeesIgnored() {
```

Comment:
FYI for capstone. Don't fix in this PR since I noticed the provided unit tests also do this.

ToTT: Writing Descriptive Test Names

> Make sure test names contain both the scenario being tested and the expected outcome